frontend/src/app/commonComponents/ComboBox/ComboBox.test.tsx

```diff
@@ -389,7 +389,7 @@ describe("ComboBox component", () => {
       );
     });
 
-    it("calls onChange prop with undefined on click", () => {
```

Comment:
One problem with changing the `ComboBox` behavior is that it is going to make it hard to migrate to the trussworks component library if we ever do so since the entire component is copypasta.